src/style.css

```
@@ -0,0 +1,144 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
```

```suggestion
* {
  box-sizing: border-box;
  margin: 0;
}
```
Goole `how selector * works` (use MDN) to find out why you can add `margin: 0` to that selector and delete all `margin: 0` below